engine/src/dpx_cid.rs

```diff
@@ -1489,14 +1485,14 @@ pub unsafe extern "C" fn CIDFont_cache_close() {
             let mut font: *mut CIDFont = 0 as *mut CIDFont;
             font = *(*__cache).fonts.offset(font_id as isize);
             if __verbose != 0 {
-                dpx_message(b"(CID\x00" as *const u8 as *const i8);
+                info!("(CID");
             }
             CIDFont_dofont(font);
             CIDFont_flush(font);
             CIDFont_release(font);
             free(font as *mut libc::c_void);
             if __verbose != 0 {
-                dpx_message(b")\x00" as *const u8 as *const i8);
+                info!(")");
```

Comment:
So this is acting like some progress indication. This use is not compatible with `log`, since newlines will be inserted for each "info!()" call.

Reply:
Maybe give `info!()` calls like this another name. Something like `spanmsg!()` ?

Reply:
Are you sure `log` requires newline for each message. Can we write own logger that doesn't add newlines for `info` or maybe `trace`?

Reply:
In the `log` framework, each `info!()` call corresponds to a single [record](https://docs.rs/log/0.4.8/log/struct.Record.html). `dpx_message` actually doesn't behave that way, it is accumulating. 

Anyway, since we're not using `log` now, i guess we can defer this discussion to the future.